City-of-Helsinki/hel-fi-drupal-grants
# feat: AU-1892: Update helsinki profile to dev version & AD role mapping

# [AU-1892](https://helsinkisolutionoffice.atlassian.net/browse/AU-1892)

## What was done

This pull request implements the AD role <-> Drupal role mapping in the `all.settings.php` file, as is suggested by the `helfi_tunnistamo` modules `.README` file. The original mapping has been removed from the `helfi_helsinki_profiili` module in [this](https://github.com/City-of-Helsinki/drupal-module-helfi-helsinki-profiili/pull/30) pull request. If you believe the mapping should be implemented in a different `*.settings.php` file, then please say so. 

## How to install

Make sure your instance is up and running on correct branch.
* `git checkout feature/AU-1892-test-hp-module`
* `make fresh`
* `make drush-cr`

## How to test

- Review the new mapping in the `all.settings.php` file. Make sure the mapping matches the old mapping in the `helfi_helsinki_profiili.settings.yml` file.
- Testing that the new mapping works can really only be done by logging into the system with an email that has an AD role connected to it. Since we don't have access to these types of accounts, testing should be done by the customer.

[AU-1892]: https://helsinkisolutionoffice.atlassian.net/browse/AU-1892?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
